Assets/MixedRealityToolkit/Providers/UnityInput/UnityTouchDeviceManager.cs

```diff
@@ -50,8 +50,10 @@ public override void Update()
                     case TouchPhase.Began:
                         AddTouchController(touch, ray);
                         break;
-                    case TouchPhase.Moved:
                     case TouchPhase.Stationary:
+                        StartTouchData(touch,ray);
```

Comment:
I'm not sure I understand this change. Why would we start the touch when the touch phase is stationary (which I think can happen multiple times during a touch) instead of the existing start when a touch controller is added?

https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5876/files#diff-11e52ac5aede93183e1852123ffc04cbL124

Reply:
I've been experimenting with this part of the code and it seems like dragging objects in the sample scene only works if the InputSystem's RaisePOinterDown event is called during TouchPhase.Stationary case in this code block.

I'm not familiar enough with MRTK to understand why but I think this is why @TakahiroMiyaura added that function call there.